mongo/integration/client_side_encryption_test.go

```diff
@@ -597,3 +597,75 @@ func TestFLE2DocsExample(t *testing.T) {
 		}
 	})
 }
+
+func TestFLE2CreateCollectionWithAutoEncryption(t *testing.T) {
```

Comment:
```suggestion
// `TestFLE2CreateCollectionWithAutoEncryption` is a regression test for a bug fixed in GODRIVER-2413.
// Prior to GODRIVER-2413, the `IndexView.CreateMany` operation was not processed for automatic encryption. This resulted in no "listCollections" command sent.
func TestFLE2CreateCollectionWithAutoEncryption(t *testing.T) {
```

Suggest describing the motivation for this test.